### DISCUSSION TITLE
请问您的主页文档是基于什么开发的？太喜欢了

### DISCUSSION THREAD

**Discussion by 8692769** [SylarLong/iztro#199] [Q&A]
请问您的主页文档是基于什么开发的？是否是基于一些开源的项目？
自己也想做个类似知识库的东西。真的太喜欢您这个风格了。

**Comment by 8692769** [comment]
不好意思，已经在您的 [iztro-docs](https://github.com/SylarLong/iztro-docs) 仓库看到了！感谢开源！谢谢您！

**Comment by SylarLong** [comment]
感谢关注！
